google-cloud-logging/lib/google/cloud/logging/service.rb

```diff
@@ -30,57 +29,53 @@ class Service
 
         ##
         # Creates a new Service instance.
-        def initialize project, credentials, timeout: nil, client_config: nil,
-                       host: nil
+        def initialize project, credentials, timeout: nil, host: nil, client_config: nil
           @project = project
           @credentials = credentials
           @timeout = timeout
           @client_config = client_config || {}
```

Comment:
Are `@client_config` and the related accessor `client_config` still used anywhere?

Reply:
Not by our code, but I left it there to avoid breaking changes.

Reply:
`Service` is an undocumented, internal class. I believe it can be removed.

Reply:
done